src/style.css

```diff
@@ -1 +1,62 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  width: 100px;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  background-repeat: no-repeat;
+  background-position: center;
+  background-image: url('images/star.svg');
+}
+
+.stars--1 .stars__star:nth-child(-n + 1) {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  background-repeat: no-repeat;
+  background-position: center;
+  background-image: url('images/star-active.svg');
+}
```

Comment:
The CSS rules for `.stars--1 .stars__star:nth-child(-n + 1)` are repeated for each `.stars--X` class with only the nth-child value changing. Consider using a more concise approach to avoid repetition, such as using a common class for active stars and adjusting the number of active stars with JavaScript or a more dynamic CSS approach.